### PR TITLE
Footer created

### PR DESCRIPTION
**Pull request by IgorBasovskyy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -69,88 +69,130 @@
...
+                    <input type="text" placeholder="Full Name">
+                    
+                    <input type="text" placeholder="Email Address">
+                    
+                    <input type="text" placeholder="Subject">
+                    
+                    <textarea name="comment" placeholder="Message"></textarea>
+                </form>
+
+                <button>SUBMIT</button>
```
Comment (uc2pac) [inline index.html]: Move button to form. Add type="submit" to button

File: styles.css [hunk 2]
```diff
@@ -198,3 +198,141 @@ header {
...
+	padding: 12px 0;
+}
+
+.links a {
+	color: #40aed7;
+	font-size: 22px;
+}
+
+.tweets {
+	width: 24%;
```
Comment (uc2pac) [inline styles.css]: Why 24% not 25%? Same for .links, .blog and .contact

File: styles.css [hunk 3]
```diff
@@ -198,3 +198,141 @@ header {
...
+form input {
+	background-color: #222222;
+	border: none;
+	height: 50px;
+	margin: 10px 0;
+	font-size: 24px;
+	color: #979797;
+	padding-left: 12px;
+	outline: none;
+	width: 97%;
```
Comment (uc2pac) [inline styles.css]: why 97%?

File: styles.css [hunk 4]
```diff
@@ -198,3 +198,141 @@ header {
...
+}
+
+form textarea {
```
Comment (uc2pac) [inline styles.css]: Please add some id to form and select it instead of form tag

File: styles.css [hunk 5]
```diff
@@ -198,3 +198,141 @@ header {
...
+	height: 125px;
+	margin: 10px 0;
+	font-size: 26px;
+	color: #979797;
+	padding: 12px 0 0 12px;
+	outline: none;
+	width: 97%;
+}
+
+button {
```
Comment (uc2pac) [inline styles.css]: Add cursor: ponter style property

File: index.html (hunk 1 above)
Comment (IgorBasovskyy) [reply index.html]: done

File: styles.css (hunk 2 above)
Comment (IgorBasovskyy) [reply styles.css]: 24% because between blocks must be empty space. If i put 25% than blocks will touch each other.

File: styles.css (hunk 3 above)
Comment (IgorBasovskyy) [reply styles.css]: inputs have padding left 12 px 
if i put them 100% width they will go out of content

File: styles.css (hunk 5 above)
Comment (IgorBasovskyy) [reply styles.css]: done

File: styles.css (hunk 4 above)
Comment (IgorBasovskyy) [reply styles.css]: don't understand

File: styles.css (hunk 3 above)
Comment (uc2pac) [reply styles.css]: you can have 100% width + padding but you need to apply box-sizing: border-box css property

File: styles.css (hunk 4 above)
Comment (uc2pac) [reply styles.css]: You are selecting textarea via form tag, like - form textarea.
I am telling you to add some id to form tag and select textarea through it, like #contact-us-form textarea

**Pull request merged**